p2p/client.go

```diff
@@ -320,9 +320,9 @@ func (c *Client) tryConnect(ctx context.Context, peer peer.AddrInfo) {
 func (c *Client) setupGossiping(ctx context.Context) error {
 	pubsub.GossipSubHistoryGossip = c.conf.GossipCacheSize
 	pubsub.GossipSubHistoryLength = c.conf.GossipCacheSize
-	pubsub.GossipSubMaxIHaveMessages = c.conf.GossipCacheSize
 
-	ps, err := pubsub.NewGossipSub(ctx, c.Host)
+	//We add WithSeenMessagesTTL (with 1 year time) option to avoid ever requesting already seen blocks
+	ps, err := pubsub.NewGossipSub(ctx, c.Host, pubsub.WithSeenMessagesTTL(8760*time.Hour))
```

Comment:
can u explain this 1 year time? previously u had 
`time.Duration(int(c.blockTime.Nanoseconds()*int64(c.conf.GossipCacheSize))))`

Reply:
i removed the ttl option and replaced gossipsub version with a new version where seen packets are not expiring